docs/guide/clients.md

```diff
@@ -57,6 +57,17 @@ See the [demo][ha-demo] for an example of the map with OwnTracks devices or read
 [ha-owntracks]: https://home-assistant.io/components/device_tracker.owntracks/
 [ha-zone]: https://home-assistant.io/components/automation/#zone-trigger
 
+
+## Owntracks-Cards
+This is a webapp to create and edit [Owntracks cards](.../features/card.md).
+It can be either used to just create the JSON representation of the card, which has to be published to the MQTT broker manually. Or it can directly publish the card to an MQTT broker connected via websockets.
+Just head over to the [demo][oc-demo], and create a card. For testing, you can connect to a public MQTT broker or configure your own broker by clicking the connection state.
```

Comment:
please remove the reference to a public broker. Throughout our documenation we strongly discourage the use of those (because of privacy reasons), so I don't want that mentioned here either.

Reply:
to clarify: on your site you can of course leave that in as it's your software; I just don't want public broker encouraged here. :)